### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by zaynekomichi** [Asprise/scannerjs.javascript-scanner-web-twain-wia-browsers-scanner.js#3]
Unable to use package in typescript project

**Comment by LowHongWei** [comment]
Hi @zaynekomichi, have you been able to use the package in TS? I was planning to use it for my project and wanted to know if it's TS friendly. Cheers!

**Comment by zaynekomichi** [comment]
No, wasnt able to. Ended up leaving the idea. Main issue is with the file system, I was building on Electron so maybe thats why it wasnt responding well
